### PR TITLE
Don't unpublish a podcast based on episode published_at

### PR DESCRIPTION
**Pull request by gcampo88**
#188

### CODE REVIEW

File: app/models/episode.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ def self.by_prx_story(story)
   end
 
   def publish_updated
-    podcast.publish_updated if podcast
+    podcast.publish_updated if podcast && published?
```
Comment (cavis) [inline app/models/episode.rb]: I sort of think you should always update the podcast here.  What if the update to this episode was that it was unpublished?

Comment (kookster) [inline app/models/episode.rb]: I agree with @ryan, also the episode shouldn't be aware of what the podcast will do with this message, it should just poke it

**Comment by kookster** [comment]
definitely a nit, but I think that would be ever so slightly better.

**Comment by gcampo88** [comment]
ok, back to you @kookster !

**Pull request merged**